locust/web.py

```diff
@@ -613,11 +614,16 @@ def update_template_args(self):
         if self.modern_ui:
             percentiles = {
                 "percentiles_to_chart": stats_module.MODERN_UI_PERCENTILES_TO_CHART,
+                "percentiles_to_statistics": stats_module.PERCENTILES_TO_STATISTICS,
             }
         else:
             percentiles = {
                 "percentile1": stats_module.PERCENTILES_TO_CHART[0],
                 "percentile2": stats_module.PERCENTILES_TO_CHART[1],
+                # Convert percentiles to percent notation for use in templates
+                "percentiles_to_statistics": [
+                    normalize_decimal(percentile * 100) for percentile in stats_module.PERCENTILES_TO_STATISTICS
+                ],
```

Comment:
I think we shouldn't have to support any new features on the legacy UI. We could leave this as is and then we wouldn't need to change the templating